### PR TITLE
Create ta.js

### PR DESCRIPTION
**Pull request by jeraldfdo**
Tamil lanuguage

[Review on Reviewable](https://reviewable.io/reviews/angular-ui/ng-grid/3610)

### CODE REVIEW

**Comment by PaulL1** [comment]
This appears to change en to have tamil decodes - I think inside the file you needed to set the language to ta?  That in turn means it breaks the unit tests.

When you fix, could you please squash the commit again into a single commit?

**Comment by c0bra** [comment]
@jeraldfdo Any change you could check out Paul's comment?

File: src/js/i18n/ta.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/**
+ * Created by Tim on 2/1/14.
+ */
+(function () {
+  angular.module('ui.grid').config(['$provide', function($provide) {
+    $provide.decorator('i18nService', ['$delegate', function($delegate) {
+      $delegate.add('en', {
```
Comment (JLLeitschuh) [inline src/js/i18n/ta.js]: This is the only line that needs to be fixed. Please change this to `$delegate.add('ta', {` and we can merge this in.

**Comment by JLLeitschuh** [comment]
Closed by #4120

**Pull request closed without merging**